Library/Formula/multimarkdown.rb

```diff
@@ -30,9 +23,15 @@ def install
       system "make"
       bin.install "multimarkdown"
     end
+
+    bin.install Dir["scripts/*"].reject { |f| f =~ /\.bat$/ }
   end
 
   test do
+    Dir["scripts/*"].reject { |f| f =~ /\.bat$/ }.each do |unix_script|
```

Comment:
`scripts/` won't exist anymore, since the buildpath is a temporary directory.  Also, can the test run one or more of these scripts, to check their functionality?